src/apps/reservation-list/modal/reservation-details/reservation-details.test.ts

```diff
@@ -613,6 +614,10 @@ describe("Reservation details modal", () => {
       fixtureFilePath: "reservation-list/work.json"
     });
 
+    cy.interceptGraphql({
+      operationName: "GetCoversByPids",
+      fixtureFilePath: "cover.json"
+    });
```

Comment:
Please consider if/why we need to add this extra intercept.

It seems as if we already intercepted the same operation in the `beforeEach()`. If we need to change something then I would suggest that we change an `cy.interceptRest()` to a `cy.interceptGraphql()` with the correct operation name.

This also applies elsewhere in this file.